### PR TITLE
Fix for issue #656

### PR DESCRIPTION
**Pull request by ffaubry**
Made aria helper more robust to arrays with null values.
Updated modal to prevent passing arrays with null values to helpers.

### CODE REVIEW

File: lib/js/helpers/aria.js [hunk 1]
```diff
@@ -18,13 +21,19 @@ export function toggleHidden (array) {
...
+      return;
+    }
     node.setAttribute('aria-hidden', true);
   });
 }
 
 // removes `aria-hidden` on a domNode
 export function show (array) {
   array.forEach(function (node) {
+    if (node) {
```
Comment (paulcpederson) [inline lib/js/helpers/aria.js]: shouldn't these be `if (!node)` to check that the node is falsy?

**Comment by ffaubry** [comment]
@paulcpederson duh! Sorry about this. I went back and forth about testing the falsy or the truthy... Ended up pushing a mess.
Please verify again.

Additional question: How do I build this for my fork?
I went thru the list of scripts.. but nothing seems safe to use...
My issue is that if we want to contribute, we should be able to build and push in our fork this way we can use the forked repo in our projects... 
Any doc on how to build the repo in forks?

**Comment by paulcpederson** [comment]
@ffaubry Im not sure I understand your question? Do you mean running the tests that travis runs locally?

**Comment by ffaubry** [comment]
@paulcpederson taking this offline.
Let me know if you can merge this PR.

**Comment by paulcpederson** [comment]
thanks for the pr, @ffaubry !

**Pull request merged**